recipes/tbpore/meta.yaml

```diff
@@ -1,44 +1,43 @@
 {% set name = "tbpore" %}
 {% set version = "0.1.0" %}
-{% set repo_URL = "https://github.com/mbhall88/tbpore/" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: {{ repo_URL }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 5e7fa6c7b25dfdbd5fbd5970ac5584c27bcd912d21eea6eeebde16f12921ff15
+  url:    https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cde34d5e17c00ee4d0f43205cf63d4c88e80dffd4cabf75a0c7c4d4c93be68e2
   
 build:
-  number: 0
+  number: 1
   noarch: python
   entry_points:
     - tbpore = tbpore.tbpore:main
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python
-    - poetry
+    - python >=3.8,<3.10
+    - poetry =1.1
     - pip
   
   run:
-    - mykrobe
+    - python >=3.8,<3.10
+    - mykrobe =0.11
     - rasusa
-    - samtools
-    - bcftools
-    - minimap2
-    - seqkit
-    - psdm
-    - importlib_metadata >=1.7,<2.0.0
-    - click
-    - cyvcf2
-    - loguru
-    - pyyaml
-    - pandas
-    - pysam
-    - networkx
+    - samtools =1.13
```

Comment:
are those strict pins really necessary? Can we loosen them?

Reply:
No, this tool's variant filters were calibrated on a very specific version (1.13) of bcftools unfortunately.

Reply:
@bgruening is that okay?